sway/desktop/output.c

```diff
@@ -307,5 +307,5 @@ void handle_new_output(struct wl_listener *listener, void *data) {
 	output->mode.notify = handle_output_mode;
 
 	arrange_layers(output);
-	arrange_windows(&root_container, -1, -1);
+	container_arrange_windows(&root_container, -1, -1);
```

Comment:
-1 to renaming this to container_arrange_windows, I don't feel like arrange_windows is a logical child of containers

Reply:
The first and only pointer is a container and it affects the state of the tree of the container. If that's not a container function, then what does it take to be a container function?